collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java

```diff
@@ -215,15 +220,24 @@ public Credentials mockGoogleCredentials() throws IOException {
     @Primary
     TraceConsumer traceConsumer(
         Credentials credentials, ZipkinStackdriverStorageProperties storageProperties)
-        throws IOException, NoSuchFieldException, IllegalAccessException {
+        throws IOException {
       final ManagedChannel managedChannel =
           NettyChannelBuilder.forTarget(storageProperties.getApiHost())
               .sslContext(CLIENT_SSL_CONTEXT)
               .build();
-      TraceServiceGrpc.TraceServiceBlockingStub traceService =
-          TraceServiceGrpc.newBlockingStub(managedChannel)
-              .withCallCredentials(MoreCallCredentials.from(credentials));
-      final GrpcTraceConsumer traceConsumer = new GrpcTraceConsumer(traceService);
+
+      TransportChannelProvider channelProvider =
+          FixedTransportChannelProvider.create(GrpcTransportChannel.create(managedChannel));
```

Comment:
so was this part the secret of the incas? (ie the part that fixed the hung test thing)

Reply:
yep, I ruled SSL out by adding main() to StackdriverMockServer and connecting to it with "openssl s_client -connect localhost:31891". Apparently new grpc server doesn't like credentials pushed unless they are expected.